Documentation/CommonDataModel_Wiki_Files/StandardizedClinicalDataTables/VISIT_OCCURRENCE.md

```diff
@@ -14,7 +14,7 @@ Field|Required|Type|Description
 |care_site_id|No|integer|A foreign key to the care site in the care site table that was visited.|
 |visit_source_value|No|string(50)|The source code for the visit as it appears in the source data.|
 |visit_source_concept_id|No|Integer|A foreign key to a Concept that refers to the code used in the source.|
-|admitting_source_concept_id|	|Integer	|No	|A foreign key to the predefined concept in the Place of Service Vocabulary reflecting the admitting source for a visit.|
+|admitting_source_concept_id|	Integer	|No	|A foreign key to the predefined concept in the Place of Service Vocabulary reflecting the admitting source for a visit.|
```

Comment:
Thanks for fixing the errant pipe. The 'Integer' and 'No' just need to be switched so that the 'No' is in the column before 'Integer'